tests/baselines/reference/nodeModules1(module=node16).js

```diff
@@ -325,51 +325,51 @@ export {x};
 Object.defineProperty(exports, "__esModule", { value: true });
 exports.x = void 0;
 // cjs format file
-var x = 1;
+const x = 1;

```

Comment:
This is a test harness fix; this file has no target set, but sets modeule=node16, which is supposed to mean es2022.